variables.tf

```diff
@@ -37,6 +37,14 @@ variable "name" {
   description = "The hosted zone name"
 }
 
+variable "dns_query_logging" {
```

Comment:
Variables are sorted. Can you move this one to the right location?

Reply:
Oh, true! Fixed it. :)